cli/Cargo.toml

```diff
@@ -15,6 +15,7 @@ path = "src/main.rs"
 iota-sdk = { path = "../sdk", default-features = false, features = [ "wallet", "tls", "storage", "rocksdb", "stronghold", "participation" ] }
 
 clap = { version = "4.2.0", default-features = false, features = [ "std", "color", "help", "usage", "error-context", "suggestions", "derive", "env" ] }
+console = { version = "0.15.5", default-features = false }
```

Comment:
This crate seems iffy. Maybe [colored](https://docs.rs/colored/latest/colored/) is better?

Reply:
`colored` is used by fern so it should already be in the dependency tree